demos/multi_material/benchmarks/schmalholz_2011.py

```diff
@@ -1,63 +1,27 @@
+from dataclasses import dataclass
+from typing import ClassVar
+
 import firedrake as fd
 import initial_signed_distance as isd
 
 import gadopt as ga
 
 
-class Mantle(ga.AbstractMaterial):
-    @classmethod
-    def B(cls):
-        return None
-
-    @classmethod
-    def RaB(cls):
-        return None
-
-    @classmethod
-    def density(cls):
-        return 3150
-
-    @classmethod
-    def viscosity(cls, velocity):
+@dataclass
+class Mantle(ga.Material):
+    def viscosity(self, *args, **kwargs):
         return 1e21
 
-    @classmethod
-    def thermal_expansion(cls):
-        return 1
-
-    @classmethod
-    def thermal_conductivity(cls):
-        return 1
-
-    @classmethod
-    def specific_heat_capacity(cls):
-        return 1
-
-    @classmethod
-    def internal_heating_rate(cls):
-        return 0
-
-
-class Lithosphere(ga.AbstractMaterial):
-    visc_coeff = 4.75e11
-    stress_exponent = 4
-    visc_bounds = (1e21, 1e25)
-
-    @classmethod
-    def B(cls):
-        return None
-
-    @classmethod
-    def RaB(cls):
-        return None
 
-    @classmethod
-    def density(cls):
-        return 3300
+@dataclass
+class Lithosphere(ga.Material):
+    visc_coeff: ClassVar = 4.75e11
+    stress_exponent: ClassVar = 4
+    visc_bounds: ClassVar = (1e21, 1e25)
```

Comment:
I think `ClassVar` needs to be instantiated with an inner type

Reply:
It seems that it is running without them, but I will still add them nevertheless :)